distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
-0.0.8 (in development)
+﻿0.0.8 (in development)
```

Comment:
You can also do it in VS, click "save as", then the save button has a drop down allowing you to select "save with encoding". From there you can set the encoding to UTF-8 without BOM.